Standards/scs-xxxx-v1-provider-network-standard.md

```diff
@@ -0,0 +1,200 @@
+Servers can be connected to multiple networks, and connecting to additional provider networks would not interfere with their ability be externally accessible.
+
+CSPs may also create multiple provider networks with different options for external access, such as separate networks for IPv4 and IPv6, or one external network for use with virtual routers and a separate shared network for direct connection.
+This mostly just adds complexity to the setup, though, as a provider network can be both external and shared at the same time, and can even provide both IPv4 and IPv6 subnets in a dual stack setup [^ds].
+
+Another problem with multiple provider networks is, that users may only be able to distinguish their respective function by their name.
+
+A single default provider network leaves no ambiguity by the user in this regard and is thus preferable from a standardisation perspective.
+
+#### Shared Provider Network
+
+A shared provider network has the benefit of being easy to use.
+Servers can be attached directly and will be accessible immediately, without the requirement to create virtual routers, project-internal networks, subnets, or floating IPs.
+
+Users do have a higher control over server ports than they have over virtual router ports, so it is important to enable Neutrons port security feature on shared provider networks to prevent spoofing.
+
+The lack of address quota may be a problem if IPv4 is used and the number of available addresses is so limited that fairness between projects needs to be enforced.
+In that case, CSPs may restrict the number of servers, routers, or ports to limit the addresses used by individual projects.
+
+#### External Provider Network and Subnet Allocation
+
+Creating a project-internal network to connect to an external provider network with a virtual router does take more effort than just using a shared network, but also offers some additional flexibility and control.
+In an internal network, users have greater control over IP allocation and may also choose to disable port security.
+With the FWaaS API extensions, they can also assign firewall rules to the virtual router, to control which traffic can pass between internal and provider networks.
+
+As described above, there are multiple methods for allocating public IP addresses to project-internal networks.
+For IPv6, the currently best option seems to be subnet allocation from a CSP-managed subnet pool, because support for Prefix Delegation is still experimental.
+For IPv4, NAT and floating IPs are generally preferred over subnet allocation because of scarcity of IPv4 address space.
+
+#### NAT and Floating IPs
+
+Subnets can only have a size that is a power of two, and will thus usually be oversized for the project.
+They also need to reserve one address for the gateway and, if DHCP is used, one or more addresses for DHCP service ports.
```

Comment:
> one or more addresses for DHCP service ports.

Is that still true with OVN? (I don't actually know, but given how OVN messes with DNS, I wouldn't be surprised if it wasn't.)

Reply:
This seems to still be true for OVN, though the owner of the port is "network:distributed" instead of "network:dhcp".
I do suspect that they have dropped support for dhcp HA though, because that would be kinda pointless.

Reply:
Okay thanks.